packages/quantic/force-app/main/default/lwc/quanticPager/e2e/quanticPager.e2e.ts

```diff
@@ -0,0 +1,95 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const numberOfResultsPerPage = 10;
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic pager ${useCase.label}`, () => {
+    test.describe('when clicking the next page button', () => {
+      test('should trigger a new search and log analytics', async ({
+        pager,
+        search,
+      }) => {
+        await pager.clickNextPageButton();
+        const response = await search.waitForSearchResponse();
+        const {firstResult} = await response.request().postDataJSON();
```

Comment:
Should not be necessary since you already awaited.

```suggestion
        const {firstResult} = response.request().postDataJSON();
```